packages/assets-controllers/src/NftController.ts

```diff
@@ -152,7 +158,8 @@ export interface NftMetadata {
   lastSale?: ApiNftLastSale;
   transactionId?: string;
   tokenURI?: string | null;
-}
+  error?: NFTFetchMetadataError | undefined;
```

Comment:
Nit: The `?` already implies that the property is sometimes not present, so we can omit it:
```suggestion
  error?: NFTFetchMetadataError;
```

Reply:
Interesting!!! Thanks!